01_Introduction_to_Citsci_Pipeline.ipynb

```diff
@@ -147,26 +147,57 @@
     "\n",
     "Supply the email associated with your Zooniverse account and project slug below. \n",
     "\n",
-    "A \"slug\" is the string of your Zooniverse username and your project name without the leading forward slash, for instance: \"username/project-name\". [Click here for more details](https://www.zooniverse.org/talk/18/967061?comment=1898157&page=1).\n"
+    "A \"slug\" is the string of your Zooniverse username and your project name without the leading forward slash, for instance: \"username/project-name\". [Click here for more details](https://www.zooniverse.org/talk/18/967061?comment=1898157&page=1).\n",
+    "\n",
+    "**If you have __not__ yet created a Zooniverse project do not provide a `slug_name` and run the subsequent cell**"
```

Comment:
I recommend we change this to:
"**The rubin.citsci package includes a feature that creates a Zooniverse project from template. If you wish to use this feature, do not provide a `slug_name` and run the subsequent cell.**"